src/Components/FilterControls.js

```diff
@@ -1,86 +1,101 @@
 import React, { Component } from 'react';
 import CardContainer from './CardContainer';
+import '../FilterControls.css';
 
 class FilterControls extends Component {
   constructor(props) {
     super(props);
     this.state = {
       location: '',
       time: '',
-      date: ''
+      date: '',
+      matchedIds: []
     }
   }
 
-  getClickedLocation = (event) => {
-    let selectedLocation = event.target.value;
+  getClickedValue = (event) => {
     this.setState({
-      location: selectedLocation
-    }, this.filterByLocation)
+      [event.target.id]: event.target.value
```

Comment:
Confused by this setState method here...it looks like you're anticipating having a state property that represents `event.target.id` --- but in your component state you only have `location`, `time`, `date`, and `matchedIds` --- I'm not sure where or what state would be updated with this as written. 🤔